xwiki-rendering-transformations/xwiki-rendering-transformation-icon/src/main/java/org/xwiki/rendering/internal/transformation/icon/IconTransformation.java

```diff
@@ -250,7 +250,7 @@ private void parseTree(List<Block> sourceBlocks)
             }
             // Look for a match in children of the source block
             List<Block> filteredSourceBlocks = this.filter.filter(sourceBlock.getChildren());
-            if (filteredSourceBlocks.size() > 0) {
+            if (!filteredSourceBlocks.size().isEmpty()) {
```

Comment:
Please build before committing.

Reply:
Sorry.
Please close the issue as false positive.